lua/autorun/server/sv_autoclean.lua

```diff
@@ -40,12 +39,13 @@ function cfcCleanServer()
     for _, entity in pairs( ents.GetAll() ) do
         if not IsValid( entity ) then continue end
 
-        local isEquippedWeapon = entity:IsWeapon() and IsValid( entity.Owner )
         local classIsBlacklisted = BlacklistedEntityTypes[entity:GetClass()]
-        if isEquippedWeapon or not classIsBlacklisted then continue end
-
-        removedCount = removedCount + 1
-        entity:Remove()
+        local isUnownedWeapon = entity:IsWeapon() and not IsValid( entity.Owner )
+        
+        if classIsBlacklisted or isUnownedWeapon then 
```

Comment:
While you're in here, can you change this to a whitelist instead? "Blacklist" doesn't make sense here

Reply:
isnt the blacklist stuff thats meant to be removed on cleanup?

Reply:
The way it was written before, anything in the blacklist would be explicitly _not_ removed

Reply:
at least, that was the intention, because turrets would technically pass the other checks but we didn't want to remove them

Reply:
could just remove the blacklist for now, a whitelist doesn't seem very useful, not much will pass the checks. only weapons that have no owner.

Reply:
just be 100% sure turrets can't be caught and we're good to go

Reply:
they wont be.